<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anya</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="rightBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#rightBar{
background:#4d4d4d;}
QTabWidget::pane{
background:#fff;
}
QTabWidget::tab-bar{
border:0;
background:#fff;
}
QTabBar::tab{
min-width:55px;
min-height:100px;
font-family:&quot;微软雅黑&quot;;
border:0;
} 
QTabBar::tab:selected{
color:#fff;
background:rgba(0,0,0,0.1)
}
QTabBar::tab:!selected{

}
QTabBar::tab:hover{

}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
padding:6px;
color:#b0b0b0;
font-family:&quot;微软雅黑&quot;;
background-color:#3f3f3f;
border:1px solid #5a5a5a;
}
QPushButton:hover{
background-color:rgba(255,255,255,0.1);
}QPushButton:pressed{
background-color:#2a2a2a;} QPushButton:checked{
background-color:#2a2a2a;
}</string>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <property name="movable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tab1">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="title">
              <string>Transform</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="tab1Widget" native="true">
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">#tab1Widget{
background:#242424;
border-left:1px solid #5a5a5a;
border-right:1px solid #5a5a5a;
}</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QWidget" name="btnWidget" native="true">
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>140</height>
                    </size>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <property name="leftMargin">
                     <number>15</number>
                    </property>
                    <property name="topMargin">
                     <number>15</number>
                    </property>
                    <property name="rightMargin">
                     <number>15</number>
                    </property>
                    <property name="bottomMargin">
                     <number>15</number>
                    </property>
                    <property name="spacing">
                     <number>10</number>
                    </property>
                    <item row="0" column="1">
                     <widget class="QPushButton" name="btnW">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>50</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                       </font>
                      </property>
                      <property name="focusPolicy">
                       <enum>Qt::StrongFocus</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>W</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QPushButton" name="btnA">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>50</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                       </font>
                      </property>
                      <property name="focusPolicy">
                       <enum>Qt::StrongFocus</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>A</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="btnS">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>50</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                       </font>
                      </property>
                      <property name="focusPolicy">
                       <enum>Qt::StrongFocus</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>S</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QPushButton" name="btnD">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>50</width>
                        <height>50</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>微软雅黑</family>
                       </font>
                      </property>
                      <property name="focusPolicy">
                       <enum>Qt::StrongFocus</enum>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>D</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                      <property name="autoExclusive">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="title">
              <string>Animation</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="tab2Widget" native="true">
                <property name="styleSheet">
                 <string notr="true">#tab2Widget{
background:#242424;
border-left:1px solid #5a5a5a;
border-right:1px solid #5a5a5a;
}</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QWidget" name="btnWidget2" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>140</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>on/off</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QWidget" name="widget_2" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <property name="spacing">
                        <number>7</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="styleSheet">
                          <string notr="true">color:#b0b0b0;</string>
                         </property>
                         <property name="text">
                          <string>T:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>80</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">QLineEdit{
font-family:&quot;微软雅黑&quot;;
color:#b0b0b0;
border:1px solid #5a5a5a;
background-color:transparent;
outline:none;
}</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QSlider" name="slider">
                         <property name="minimumSize">
                          <size>
                           <width>150</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>16777215</height>
                          </size>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">QSlider::groove:horizontal {
height:6px;
background-color:#3f3f3f;
}
QSlider::handle:horizontal {
width:18px;
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(124, 124, 124), stop: 1.0 rgb(72, 71, 71));
margin:-6px 0px -6px 0px;
border-radius:9px;
}</string>
                         </property>
                         <property name="minimum">
                          <number>10000</number>
                         </property>
                         <property name="maximum">
                          <number>300000</number>
                         </property>
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="GLWidget" name="openGLWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>556</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
